gtest/src/manager.rs

```diff
@@ -598,8 +598,8 @@ impl ExtManager {
         )
     }
 
-    pub(crate) fn mint_to(&mut self, id: &ProgramId, value: Balance) {
-        if value < crate::EXISTENTIAL_DEPOSIT {
+    pub(crate) fn mint_to(&mut self, id: &ProgramId, value: Balance, keep_alive: bool) {
```

Comment:
Perhaps you don't know yet that I personally don't like primitive types especially booleans =) .

This remark is not sufficient but just to mention. Better introduce the new enumuration, for example `MintMode` with two variants `KeepAlive`, `AllowDeath`. Then calls to the method would look like `self.mint_to(&value_destination, balance, MintMode::AllowDeath/KeepAlive);` instead of not much informative `self.mint_to(&value_destination, balance, false/true);`